src/vs/workbench/common/editor/sideBySideEditorInput.ts

```diff
@@ -142,6 +142,12 @@ export class SideBySideEditorInput extends EditorInput implements ISideBySideEdi
 
 		return false;
 	}
+
+	override toUntyped(group: GroupIdentifier | undefined, context: UntypedEditorContext): IUntypedEditorInput | undefined {
```

Comment:
This seems weird to me, I do not think we support such a untyped side by side editor input? The closest is this:

https://github.com/microsoft/vscode/blob/97d770d4edf9d8ed44d02ab2b1cef284e6d1dc29/src/vs/workbench/common/editor/diffEditorInput.ts#L140-L154

Reply:
If so, should the `diffEditorInput` call `super.toUntyped`?

Reply:
Or should we create an untyped side by side input?

Reply:
There is too many things going on here at once, I will remove this overload and follow up in https://github.com/microsoft/vscode/issues/127131